packages/framework/ai-collab/src/test/ai-collab/jobBoardBenchmark.spec.ts

```diff
@@ -342,6 +347,9 @@ describe.skip("AI Job Listings App Benchmark", () => {
 		}
 		const foundJohnDoe = createJohnDoeCandidateTask.data;
 		assert(foundJohnDoe !== undefined);
+		if ("diffs" in response) {
+			assert(response.diffs !== undefined);
+		}
```

Comment:
Instead of a check if the property exists, we can do something more type-safe. Further up, right after getting the response, I think something like this would ensure that for the rest of the test, `response` has the expected type:

```typescript
if (response.status !== "success") {
  throw new Error("Received an error response from ai collab");
}
```

Also, can we check the contents of the diffs? That would make for a better test.

Reply:
Make sense! Just verifying it's success response here